### PR TITLE
New events page

### PR DESCRIPTION
**Pull request by JohnyTheCarrot**
This implements the new Events page per Kristin's design.

### CODE REVIEW

**Review by karangurjar — APPROVED** [review]

File: web/client/src/components/EarthquakeList/index.js [hunk 1]
```diff
@@ -7,61 +7,35 @@ import {
...
-            </StructuredListCell>
-            <StructuredListCell>
-              <div className="earthquake-list__item-loc">
-                10 mi S of Oaxaca, Mexico
-              </div>
-              <div className="earthquake-list__item-time">
-                01:45:03 (CST) Nov 15, 2020
-              </div>
-            </StructuredListCell>
+          {range(5).map((n) => (
```
[the remark this answers is not in the document]
Comment (JohnyTheCarrot) [reply web/client/src/components/EarthquakeList/index.js:20]: I don't think the API is ready yet, no.

[the remark this answers is not in the document]
Comment (rdkelley) [reply web/client/src/components/EarthquakeList/index.js:20]: The whole front end is hard coded so far, but that's going to change soon -- now that we have sensor data available from the sensor onboarding process, we can start building a way to bring that down to the client. I'm going to start working out some of the preliminary work for that today, and hopefully during our dashboard meeting we can start creating some issues that people can work on to get us there.

[the remark this answers is not in the document]
Comment (karangurjar) [reply web/client/src/components/EarthquakeList/index.js]: cool.

**Review by karangurjar — APPROVED** [review]
Looks good to me, we can work later to inject real data once API is ready.

**Comment by JohnyTheCarrot** [comment]
Bringing in an extension on this PR. It'll be a large extension I'm afraid.

**Pull request merged**